GameServerLib/Logic/Packets/PacketNotifier.cs

```diff
@@ -238,6 +238,8 @@ public void NotifyLevelUp(Champion c)
         public void NotifyRemoveItem(Champion c, byte slot, byte remaining)
         {
             var ri = new RemoveItem(c, slot, remaining);
+            c.Spells[(byte)(slot + 6)].DeactivateSpell();
+            c.Spells[(byte)(slot + 6)] = new GameObjects.Spell(c, "", (byte)(slot + 6));
```

Comment:
These two lines look pretty hacky and are not commented in any way. Comment them at the very least.